__tests__/integration/one-app.spec.js

```diff
@@ -638,7 +638,7 @@ describe('Tests that require Docker setup', () => {
         await expect(searchForRequerstLog).resolves.toMatch(requestLogRegex);
       });
 
-      it('log gets updated when Tenancy Root module gets updated', async () => {
+      it('log gets updated when Application Root module gets updated', async () => {
```

Comment:
what do you think about changing this to only _Root module_  we might not need the Application part

Reply:
sounds good!